frontend/taipy-gui/src/components/Taipy/AlertComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback, useMemo } from "react";
+import Alert from "@mui/material/Alert";
+import { TaipyBaseProps } from "./utils";
+import { useClassNames } from "../../utils/hooks";
+
+interface AlertComponentProps extends TaipyBaseProps {
+    severity?: "error" | "warning" | "info" | "success";
+    message: string;
```

Comment:
This should be dynamic string 
You can check in others component how dynamic strings are handled